geo-types/src/multi_point.rs

```diff
@@ -91,6 +98,78 @@ impl<T: CoordinateType> MultiPoint<T> {
     }
 }
 
+impl<T: CoordinateType> MultiPoint<T> {
+    /// Return the number of coordinates in the `MultiPoint`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use geo_types::MultiPoint;
+    ///
+    /// let mut coords = vec![(0., 0.), (5., 0.), (7., 9.)];
+    /// let multi_point: MultiPoint<f32> = coords.into_iter().collect();
+    /// assert_eq!(3, multi_point.num_coords());
+    /// ```
+    pub fn num_coords(&self) -> usize {
```

Comment:
This is a helpful method. 

Since @frewsxcv has a WIP implementation for the same functionality here: https://github.com/georust/geo/pull/563, I think it'd be better to avoid the public API churn - can you make this method private or `pub(crate)` for now, and add a comment linking to the as-of-yet unmerged PR to explain why it's not public.

Reply:
BTW https://github.com/georust/geo/pull/563 has since merged.

Could you please rebase or merge against master and use the newly added `CoordsIter#coords_count` method instead of this one?

Reply:
There is a question of placement, I would like to discuss..

CoordsItrer#coords_count is defined in the geo module.

And there is a security conflict. The implementation of the trait must be in the module defining the trait or the module containing the struct upon which the trait operates. 

The conflict is that I can't 

A) use coords_count() in geo_types.

or 

B) I can't define RelatveEq for Point<T> in geo because it it viewed as a unsafe third party module.

I will post and update when I have found it ... but I can't do what you ask.

Reply:
So I have rebased and resolved the deprection notices. here 

"After rebase, drop the use of the deprecated num_coords() method."

The solution is that coords_count is a simple one-liner - so I opted to break the DRY principle and inline the function is a few places .. no biggie

Reply:
It think this will lead to frustration for our users down the road to have methods with identical functionality but different names. I don't think people will be able to remember that  *all* geo::Geometries can use `coords_count` but that only *some* geo::Geometries can use `num_coords`. 

As you rightly point out, we unfortunately can't use the `geo::CoordsIter` trait in the `geo-types` crate, so instead I think it's best to delete this `num_coords` method and use the more verbose `self.0.len()` in the `geo-types` crate.